docs/04-publishing-deployment/getting-started.md

```diff
@@ -9,34 +9,8 @@ Running DSP-API locally or on a server requires [Docker](https://www.docker.com)
 can be freely downloaded. Please follow the instructions for installing
 [Docker Desktop](https://www.docker.com/products/docker-desktop).
 
-Additional software:
```

Comment:
not sure why we would have a "getting started" file under "publishing and deployment"... but that's probably out of scope for this task.  
But IMO this file could be removed entirely.

Reply:
You are right, I removed it.